readthedocs/rtd_tests/tests/test_profile_views.py

```diff
@@ -35,23 +35,6 @@ def test_edit_profile(self):
         self.assertEqual(self.user.last_name, 'Docs')
         self.assertEqual(self.user.profile.homepage, 'readthedocs.org')
 
-    def test_edit_profile_with_invalid_values(self):
```

Comment:
We should keep the tests

Reply:
I have added the tests.
Just one doubt, `max_length` is already well-tested feature of django, then what was the need for the these tests ?

Reply:
Just to be sure we don't break this in the future

Reply:
Okay...
Thanks..!!